astro/src/content/docs/extend/examples/modeling-organizations.mdx

```diff
@@ -214,23 +244,39 @@ Here's an example of the response from the [Retrieve Grants](/docs/apis/entities
 }
 ```
 
-This response indicates that the User belongs to 3 organizations and has various permissions for each. To manage this in your AwesomeCRM application, you can present an option to the user when they log in, to select the company they'd like to work on initially. This option screen might look like this:
+This response indicates that the user belongs to three organizations and has different permissions for each. To manage this in your AwesomeCRM application, you can provide users with an option to select the company they would like to work on when they log in, as in the example below.
```

Comment:
In the example app, richard is the user with multiple companies.
![Screenshot 2024-09-12 at 4 42 39 PM](https://github.com/user-attachments/assets/c2b372b4-b513-4c38-999b-0da49858a405)

Reply:
I have updated the image